packages/enzyme-test-suite/test/Utils-spec.jsx

```diff
@@ -693,6 +700,45 @@ describe('Utils', () => {
     });
   });
 
+  describe('validEmptyValues', () => {
+    describe('when recieves "false" or null', () => {
+      it('returns true', () => {
+        const validValues = [false, null];
+
+        validValues.forEach((value) => {
+          expect(validEmptyValues(value)).to.equal(true);
+        });
+      });
+    });
+
+    describe('when it recieves any other value than "false" or null', () => {
+      it('returns false', () => {
+        const values = [true, 'test', []];
+
+        values.forEach((value) => {
+          expect(validEmptyValues(value)).to.eql(false);
+        });
+      });
+    });
+  });
+
+  describe('renderedDive', () => {
+    const emptyNodetestData = emptyNodes();
+    const nonEmptyNodeData = nonEmptyNodes();
+
+    itWithData(emptyNodetestData, 'when renderedDive receives nodes that render validEmpty values it returns: ', (data) => {
```

Comment:
i'm not sure i'm a fan of generative test cases - i'd rather have them all laboriously written out, or, a hardcoded test that iterates through a hardcoded array of data.

Reply:
Sounds good, I had originally done that but saw the generative test pattern so I went with that.  Ill fix.